cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-cluster-api/cluster-api-controller:v0.1.4
+      - image: us.gcr.io/k8s-artifacts-prod/cluster-api/cluster-api-controller:v0.1.9
```

Comment:
intended to swtich to us.gcr?

Reply:
@sbueringer I do not know the exact reason, see kubernetes-sigs/cluster-api#1258

Reply:
Ah okay, good enough :)